client/state/reader/teams/actions.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal deps
+ */
+import wpcom from 'lib/wp';
+import {
+	READER_TEAMS_REQUEST,
+	READER_DATA_REQUEST,
+} from 'state/action-types';
+
+const wpcomUndocumented = wpcom.undocumented();
+
+export function requestTeams() {
+	return {
+		type: READER_DATA_REQUEST,
+		requestAction: READER_TEAMS_REQUEST,
+		dataFetch: wpcomUndocumented.readTeams.bind( wpcomUndocumented ),
```

Comment:
Please see my comment above about passing a function through an action. It can have serious unintended consequences that break the Redux flow.